### ISSUE TITLE
An accessibility issue with the "Views" landing page

### ISSUE THREAD

**Issue by cindyli** [inclusive-design/wecount.inclusivedesign.ca#328]
## Describe the bug

Lighthouse report shows expand buttons on the "Views" landing page are missing names.

## To Reproduce

1. Go to https://dev--wecount.netlify.app/views/;
2. Run lighthouse report;
3. The accessibility score drops from 100 to 92;
4. The accessibility section below reports expand buttons are missing names.

## Expected behavior

1. Expand buttons should have proper names after they perform expand or collapse actions. 
2. The lighthouse score for accessibility for this page should be 100.

## Screenshots

<img width="756" alt="Screen Shot 2020-07-16 at 11 52 26 AM" src="https://user-images.githubusercontent.com/572877/87693336-d8698700-c75a-11ea-87bd-c61e5f6dba29.png">
